Jenkinsfile

```diff
@@ -125,16 +121,14 @@ def CloudInstallAndTest(cloudTarget) {
   def nodeReq = "ubuntu && amd64 && ${cloudTarget}"
   node(nodeReq) {
     echo "Installing DLR package for ${cloudTarget} target"
-    withAWS(credentials:'Neo-AI-CI-Fleet') {
-      files = s3FindFiles(bucket: 'neo-ai-dlr-jenkins-artifacts', glob: "${env.JOB_NAME}/${env.BUILD_ID}/artifacts/dlr-*-manylinux1_x86_64.whl")
-      assert files.size() == 1
-      files.each {
-        s3Download file: it.name, bucket: 'neo-ai-dlr-jenkins-artifacts', path: it.path, force: true
-      }
+    if (cloudTarget == "p2" || cloudTarget == "p3") {
+      unstash 'dlr_gpu_whl'
+    } else {
+      unstash 'dlr_cpu_whl'
     }
     sh """
-    ls -lh *.whl
-    pip3 install *.whl
+    ls -lh python/dist/*.whl
+    pip3 install python/dist/*.whl
     pip3 install tensorflow
```

Comment:
the command to install tf for gpu is `pip3 install tensorflow_gpu`

Reply:
change to tensorflow_gpu for p3 and p2